xray/client.go

```diff
@@ -117,3 +118,8 @@ func (rt *roundtripper) RoundTrip(r *http.Request) (*http.Response, error) {
 	})
 	return resp, err
 }
+
+func getURLMinusQuery (u url.URL) string {
+	u.RawQuery = ""
+	return u.String()
+}
```

Comment:
Can we format this file ? (as per reviewdog comment)

Reply:
Formatted